ApptentiveFeedback/ApptentiveFeedback/Transforms/Metadata.xml

```diff
@@ -0,0 +1,40 @@
+﻿<metadata>
+
```

Comment:
Noticed this LogLevel being defined in Apptentive Core module's metadata as well. Is it necessary to have it in both places?